network/book.go

```diff
@@ -219,7 +219,7 @@ func (s *SimpleBook) Sample() ([]string, error) {
 func (s *SimpleBook) slice(active bool) []*entry {
 	entries := make([]*entry, 0)
 	for _, e := range s.entries {
-		if !active && e.active {
+		if e.active != active {
```

Comment:
Maybe we can rename param to includeActive instead? This would improve readability.